pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <!-- POM version -->
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- Artifact description -->
+    <groupId>com.stelesoft</groupId>
+    <artifactId>fortify</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>FortiFY</name>
+    <url>http://www.stelesoft.com</url>
+
+    <!-- Load IDE-specific settings (from parent project) -->
+    <parent>
+        <groupId>com.stelesoft</groupId>
+        <artifactId>java-ide-settings</artifactId>
+        <version>[0.1.0-SNAPSHOT,2.0.0)</version>
+        <relativePath>./java-settings/pom.xml</relativePath>
+    </parent>
+
+    <!-- Global build properties -->
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <javafx.version>11.0.2</javafx.version>
```

Comment:
Using 11.0.2 matches our initial plan, but might be too restrictive. It doesn't work for applications that utilize a newer version of JavaFX. This could happen if they're creating in-house builds or getting newer builds from Gluon. End-user applications could end up dependent on 2 different versions of JavaFX, which would be bad. 

**FIX**: Make this a version range. We should keep it within the same major version, so probably something like:
```[11.0.2,12.0.0)```

NOTE: Past version of JavaFX have had functional incompatibilities within a major version, so this may not be good enough. We might have to further restrict the max value in the future. However, restricting to the major version is a good place to start